DOC-1684/public-documentation-account-admin-change (#616)
swan-io/docs: docs/topics/accounts/memberships/admin-change.mdx

---
title: Account administrator change
description: Change the legal representative for a company account holder with a structured review process, supporting document collection, and identity verification.
---

> # Account administrator change

Reviewer: ```suggestion
# Change account administrator
```
Verb led headings

> When a company needs to change the person who administers its Swan accounts, Swan provides a structured process called an **account administrator change**. This process collects information and documents about the new administrator, reviews them, and promotes the new administrator on all of the company's accounts once approved.

Reviewer: Lead with the purpose of the page. 
```suggestion
Change the legal representative who administers a company's Swan accounts. Swan reviews the request, collects documents, and promotes the new administrator across all of the account holders' accounts.
```

:::info
Account administrator changes are only available for **company** account holders. Individual accounts and self-employed account holders aren't eligible.
:::

## Overview

An account administrator change applies to **all accounts** belonging to the same account holder. There can only be one ongoing change per account holder at a time.

The form used to submit the request is **unauthenticated**, meaning the requester doesn't need to be an existing account member. This allows third parties, such as a company secretary, to submit the request.

> ## Process {#process}

Reviewer: Meaningful heading - "How to change account administrator?"

> ## Process {#process}

Author: ```suggestion
## Walkthrough {#process}
```

### Step 1: Initiate the request {#initiate}

> Contact Swan's support team via a **support ticket** with the account holder information (IBAN or account holder name).

Reviewer: ```suggestion
Submit a [Swan Support request](https://supportform.swan.io/) with the account holder information (IBAN or account holder name).
```
Link the partner support form

> - This process is not yet self-service in the Dashboard. All requests go through Swan support to keep all exchanges in a single Zendesk ticket.

Reviewer: ```suggestion
- This process is not yet self-service in the Dashboard. All requests go through Swan Support to keep all exchanges in a single Zendesk ticket.
```

> - Swan processes the request, creates the account administrator change, and generates a **form URL**.

Reviewer: We usually say "link" vs "form URL"

- Swan shares the form URL with you (the partner) to forward to the requester. Partners can't generate this URL themselves.
- The form URL is valid for **7 calendar days**. After expiry, the status moves to `Expired`, the supporting document collection is `Canceled`, and a new request must be initiated.

### Step 2: Submit the request form {#submit-form}

The requester opens the form URL and submits the following information:

- Requester details (if the requester is different from the new administrator).
- New account administrator details.
- Reason for the change.
- Required [supporting documents](#documents).

Because the form is unauthenticated, any third party can fill it (for example, a company secretary or an outgoing legal representative who no longer has account access).

### Step 3: Swan reviews the request {#review}

> After submission, the status moves to `Pending`. Swan's KYC team reviews the submitted documents.

Reviewer: ```suggestion
After submission, the status moves to `Pending`. Swan's Know Your Customer (KYC) team reviews the submitted documents.
```

> After submission, the status moves to `Pending`. Swan's KYC team reviews the submitted documents.

Reviewer: Also since we're in the Swan world - Swan doesn't need to be repeated so many times.

> - If additional information is needed, the status moves to `WaitingForInformation`. Swan generates a new dedicated document upload URL and shares it with you via the support ticket. The original form URL can no longer be used at this stage.

Reviewer: ```suggestion
- If additional information is needed, the status moves to `WaitingForInformation`. Swan generates a new dedicated document upload URL and shares it with you through the same support ticket. The original form URL cannot be used at this stage.
```

> - If additional information is needed, the status moves to `WaitingForInformation`. Swan generates a new dedicated document upload URL and shares it with you via the support ticket. The original form URL can no longer be used at this stage.

Reviewer: ```suggestion
- If additional information is needed, the status moves to `WaitingForInformation`. Swan generates a new dedicated document upload URL and shares it with you via the support ticket. The requester can no longer use the original form link at this stage.
```

Active

- After the additional documents are uploaded, the status returns to `Pending` and the review resumes.

### Step 4: New administrator receives an invitation {#invitation}

After documents are validated, Swan invites the new administrator to join all accounts belonging to the account holder. The invitation grants full permissions: `canViewAccount`, `canInitiatePayments`, `canManageAccountMembership`, `canManageBeneficiaries`, and `canManageCards` (all `true`).

> After accepting the invitation, the new administrator is automatically directed to Swan's identity verification flow. No manual link sharing is needed.

Author: ```suggestion
After accepting the invitation, the new administrator is automatically directed to Swan's [identity verification flow](/topics/users/identifications/). No manual link sharing is needed.
```

> After accepting the invitation, the new administrator is automatically directed to Swan's identity verification flow. No manual link sharing is needed.

Reviewer: Link to identity flow content.

:::tip Invitation not received?

> If the new administrator doesn't receive the invitation, you can resend it using the `sendAccountMembershipInviteNotification` mutation (up to 5 times per day, status must be `InvitationSent`), or contact Swan support via the support ticket.

Reviewer: We don't use "via"

```suggestion
If the new administrator doesn't receive the invitation, you can resend it using the `sendAccountMembershipInviteNotification` mutation (up to 5 times per day, status must be `InvitationSent`), or submit a Swan Support request. 
```

> If the new administrator doesn't receive the invitation, you can resend it using the `sendAccountMembershipInviteNotification` mutation (up to 5 times per day, status must be `InvitationSent`), or contact Swan support via the support ticket.

Reviewer: ```suggestion
If the new administrator doesn't receive the invitation, you can resend it using the `sendAccountMembershipInviteNotification mutation`. The membership status must be InvitationSent, and you can send up to 5 invitations per day, or contact Swan support via the support ticket.
```

Split it out for clarity.

:::

### Step 5: New administrator is promoted {#promotion}

> Before promotion, the new administrator must have the required identification level for their IBAN country.

Reviewer: Why do we call this promotion?

| IBAN country | Required identification level |
| --- | --- |
| France, Belgium, Italy, Netherlands, Spain | Expert |
| Germany | QES (Qualified Electronic Signature) |

What happens next depends on the new administrator's current identification level:

> - **No prior identification:** Swan guides the new administrator through the appropriate identification process automatically after they accept the invitation.

Author: ```suggestion
- **No prior identification:** Swan guides the new administrator through the appropriate [identification](/topics/users/identifications/) process automatically after they accept the invitation.
```

> - **No prior identification:** Swan guides the new administrator through the appropriate identification process automatically after they accept the invitation.

Reviewer: Link to the identification content.

- **Lower identification level than required:** Swan support reaches out with a link to complete the appropriate process. The status remains `InvitationSent` until the required level is achieved.
- **Already has the required level:** The process proceeds immediately to promotion.

Once the account membership is `Enabled` on all accounts, Swan promotes the new administrator as the account administrator. The status moves to `Verified` and the supporting document collection moves to `Approved`.

### Step 6: Clean up account access {#clean-up}

> The previous administrator is **not** automatically removed or downgraded. Their membership and permissions remain unchanged.

Reviewer: ```suggestion
Swan doesn't automatically remove or downgrade the previous administrator. Their membership and permissions remain unchanged.
```

Active

The new administrator should review the previous administrator's access and decide how to proceed:

- [Disable the membership](./guide-disable.mdx) entirely.
- Keep the membership with full permissions.
- [Update the membership](./guide-update.mdx) to a limited role.

> ## Eligibility

Reviewer: This needs to be moved up - the requester should know if they're eligible before they start the process.

- **Requester is different from the new administrator:** The requester provides both their own details and the new administrator's details.

## Canceling a request {#cancel}

Account administrator change requests can't be canceled directly via the API or Dashboard. You have two options:

- **Wait for expiry:** If the status is still `Ongoing`, the form URL expires automatically after 7 calendar days.
- **Contact Swan support:** If the status is `Pending` or `WaitingForInformation`, contact Swan support via the support ticket to request a refusal.

After expiry or refusal, a new request can be initiated.

## Limitations {#limitations}

| Limitation | Description |
| --- | --- |
| Account holder type | Only company account holders are eligible. |
| Concurrent requests | Only one active account administrator change per account holder at a time. |
| Form expiry | Form URL expires after 7 calendar days. |
| Scope | The change applies to all accounts under the same account holder. |
| Initiation | No self-service yet. Must be requested via a Swan support ticket. |

> | Country-specific fields | The form doesn't collect residency address or tax identification number for DE, NL, and IT accounts. Partners must collect them separately. |

Reviewer: Spell out the country names if it's the first reference to them on the page. Make it consistent throughout

> | Identity verification level | QES required for DE accounts. Expert required for FR, BE, IT, NL, and ES accounts. |

Author: ```suggestion
| Country-specific fields | The form doesn't collect residency address or tax identification number for Germany, Netherlands, and Italy accounts. Partners must collect them separately. |
| Identity verification level | QES required for Germany accounts. Expert required for France, Belgium, Italy, Netherlands, and Spain accounts. |
```

## API reference

:::info

> The API operations listed on this page are coming soon. They'll be available starting **13 April 2026**.

Reviewer: Is this live or currently in sandbox?

Author: Confirmed still @comingSoon in the schema as of today (April 9).
